docs/source/tracking.rst

```diff
@@ -132,6 +132,11 @@ server running, the MLflow client interacts with the tracking server via REST re
 
 .. figure:: _static/images/scenario_3.png
 
+.. code-block:: bash
+    :caption: Command to run the tracking server in this configuration
+
+    mlflow server --backend-store-uri file::///path/to/mlruns --no-serve-artifacts
```

Comment:
Separated the command from the image to make it easier to edit and copy.

Reply:
Fantastic!

```suggestion
.. code-block:: bash
    :caption: Command to run the tracking server in this configuration

    mlflow server --backend-store-uri file:///path/to/mlruns --no-serve-artifacts
```

Reply:
Thanks for the catch!